1_task/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <script src="1_task.js"></script>
```

Comment:
basically it's not a good practise t name files with number
name should say what code does